tests/io/qchem/test_outputs.py

```diff
@@ -492,6 +501,37 @@ def test_nbo_3_c(self):
         assert perturb_ene[0]["perturbation energy"][3209] == 3.94
 
 
+def test_gradient(tmpdir):
```

Comment:
I spoke to a `pytest` developer once who said `tmpdir` is slated for eventual deprecation so better to use `tmp_path` here.

Reply:
@janosh: Thanks! Didn't know about that. I just changed them out. Resolving this.

Reply:
Ew, that's gross. I forgot since it's `Path` now, it doesn't have that. Changed. Hopefully this works.